src/include/uapi/ipc/topology.h

```diff
@@ -65,6 +65,7 @@ enum sof_comp_type {
 	SOF_COMP_KEYWORD_DETECT,
 	SOF_COMP_KPB,			/* A key phrase buffer component */
 	SOF_COMP_SELECTOR,		/**< channel selector component */
+	SOF_COMP_DEMUX,
```

Comment:
Please check this is aligned with kernel.

Reply:
When pushing this pr, kernel and fw we're not yet aligned on same versioning. I will make the update when this is done, I don't know how to do it in the middle of another version